packages/server-side-graphql-client/tests/index.test.js

```diff
@@ -0,0 +1,225 @@
+const { Text, Integer } = require('@keystonejs/fields');
+const { multiAdapterRunners, setupServer } = require('@keystonejs/test-utils');
+
+const {
+  createItems,
+  createItem,
+  deleteItem,
+  deleteItems,
+  getItemById,
+  getItems,
+  updateItem,
+  updateItems,
+} = require('../lib/server-side-graphql-client');
+
+const testData = [
+  {
+    data: {
+      name: 'test',
+      age: 30,
+    },
+  },
+  { data: { name: 'test2', age: 40 } },
+];
+const schemaName = 'testing';
+
+const seedDb = ({ keystone, ...rest }) =>
```

Comment:
We never use `...rest` so it should be removed.

Reply:
I use it! What do you want to name it? options?